### ISSUE TITLE
Howling Wolf/White Tiget flavor text

### ISSUE THREAD

**Issue by TheOrgg** [buttonmen-dev/buttonmen#2900]
White Tiger is fast, extremely aggressive.  He will not stop till he has beaten all competition up.

Howling Wolf is playful yet viciously fierce.  A newcomer on the scene yet has years of experience.

...whatever that means.  

That looks like all the flavor texts from the rarer sets I obtained from Rhea Shelley.

**Comment by TheOrgg** [comment]
@blackshadowshade , just noticed this hadn't been hit.  Flavor text was one that was really easy to fix, right?

**Comment by blackshadowshade** [comment]
Yes, you're correct. Thanks for bumping this, I'll get onto it soon.

**Comment by TheOrgg** [comment]
@blackshadowshade , noticed when checking the text file vs current flavor text that these two hadn't been updated yet.
